### DISCUSSION TITLE
How to get stroke-width on v2 to match v1?

### DISCUSSION THREAD

**Discussion by dmoKruso** [tailwindlabs/heroicons#837] [Help]
As the title says, how can I get stroke-width on v2 to match v1? I saw it was 2px and now it's 1.5px.
I am trying to figure out the best course of action to achieve it.

**Comment by whitespacecode** [comment]
I made a dynamic <Heroicon> component so i only needed to add this once, but there i added a class to the icon so the stroke would be 2px

`<component v-if="heroIcon" :is="heroIcon" class="[&>path]:stroke-[2]"/>`

This also works if you are adding the icons by yourself
`<BeakerIcon className="h-6 w-6 text-blue-500 [&>path]:stroke-[2]"/>`

Or just adding general css rules could also work
```
.stroke-1 path{
   stroke-width: 1;
}
.stroke-2 path{
   stroke-width: 2;
}
```

**Comment by adamwathan** [comment]
If you prefer the look of v1 I would recommend just using v1. The v2 icons are designed with a 1.5px stroke in mind and will look worse if you change the stroke to 2px.
